### PR TITLE
4.4.1 Release

### PR DESCRIPTION
**Pull request by lcharette**

### CODE REVIEW

File: app/sprinkles/core/src/Twig/CoreExtension.php [hunk 1]
```diff
@@ -49,20 +53,20 @@ public function getName()
     /**
      * Adds Twig functions `getAlerts` and `translate`.
      *
-     * @return array[\Twig_SimpleFunction]
+     * @return array[TwigFunction]
```
Comment (Silic0nS0ldier) [inline app/sprinkles/core/src/Twig/CoreExtension.php:56]: Fairly sure the correct type annotation is `TwigFunction[]` here. Current format indicates an array of mixed values where keys are of type `TwigFunction`.

File: app/sprinkles/core/src/Twig/CoreExtension.php [hunk 2]
```diff
@@ -73,7 +77,7 @@ public function getFunctions()
     /**
      * Adds Twig filters `unescape`.
      *
-     * @return array[\Twig_SimpleFilter]
+     * @return array[TwigFilter]
```
Comment (Silic0nS0ldier) [inline app/sprinkles/core/src/Twig/CoreExtension.php:80]: As before. `TwigFilter[]`

File: app/sprinkles/core/src/Twig/CoreExtension.php (hunk 1 above)
Comment (lcharette) [reply app/sprinkles/core/src/Twig/CoreExtension.php:56]: I'll have to look at the doc, but both seams accepted by my IDE ¯\_(ツ)_/¯

Comment (Silic0nS0ldier) [reply app/sprinkles/core/src/Twig/CoreExtension.php:56]: IDE type analysers are rather inconsistent here, speaking from experience (3 different VSCode PHP intellisense plugins handled array syntax differently, the spec way at least was consistent).
https://docs.phpdoc.org/latest/guides/types.html#arrays

Comment (Silic0nS0ldier) [reply app/sprinkles/core/src/Twig/CoreExtension.php:56]: Just to add, one of the plugins treated the above syntax like it indicates the array key type. Which frankly is rather neat and would be nice to see become a standard.

Comment (lcharette) [reply app/sprinkles/core/src/Twig/CoreExtension.php:56]: I've used `array<key,value>` in my Sprinkle. Works with VSCode Intelephense.

Comment (lcharette) [reply app/sprinkles/core/src/Twig/CoreExtension.php:56]: Anyway, I guess it can wait for later. More in those class that need to be fixed...

Comment (lcharette) [reply app/sprinkles/core/src/Twig/CoreExtension.php:56]: https://github.com/php-fig/fig-standards/blob/211063eed7f4d9b4514b728d7b1810d9b3379dd1/proposed/phpdoc.md#collections

Comment (lcharette) [reply app/sprinkles/core/src/Twig/CoreExtension.php:56]: (Apparently that got rejected somehow...)

Comment (Silic0nS0ldier) [reply app/sprinkles/core/src/Twig/CoreExtension.php:56]: Fair enough.

Re the proposal, kinda glad about the outcome. I'd rather that syntax be reserved for generics, a much more impactful feature.

File: app/sprinkles/core/src/Twig/CoreExtension.php (hunk 2 above)
Comment (Silic0nS0ldier) [reply app/sprinkles/core/src/Twig/CoreExtension.php:80]: As per above thread, this is to be followed up at a later time.

**Pull request merged**